### DISCUSSION TITLE
Unable to turn off autocomplete suggestions in python code

### DISCUSSION THREAD

**Discussion by vsfeedback** [Microsoft/PTVS#7334] [Enchancements]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/Unable-to-turn-off-autocomplete-suggesti/1650816)._

---
I am unable to see a way to turn off autocomplete suggestions for my python project. I have gone into Tools > options > text editor > Python and there is no option to turn off autocomplete suggestions in any of the tabs. Are there any alternative options to turn off this feature?

---
#### Feedback Bot on 2/2/2022, 05:23 PM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

---
### Original Solutions
(no solutions)

**Comment by AdamYoblick** [comment]
I looked at the options for other languages (C#, etc...) and they do seem to support options that limit intellisense and autocomplete. Pylance doesn't currently support this, so maybe that's why it's not there. Needs more investigation on our end.

**Comment by StellaHuang95** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.

**Comment by AdamYoblick** [comment]
There is another issue which I'm closing as a duplicate of this one, but it also mentions disabling intellisense in the interactive window as well. So we should have the switch control intellisense in both the editor and the repl window.
